### DISCUSSION TITLE
nintendo switch port ?

### DISCUSSION THREAD

**Discussion by pramates** [unknownskl/xbox-xcloud-client#931] [Ideas]
Would it be possible port greenlight to nintendo switch ?

**Comment by gunnaraas** [comment]
Unlikely. Greenlight provides a provides a way to interact with the [Xbox Remote Play](https://www.xbox.com/en-US/consoles/remote-play) feature. 

The Switch doesn't have any way to host a remote play session - and frankly, it likely lacks the processing power and network bandwidth to do it well (though [some homebrew implementations](https://github.com/exelix11/SysDVR) do exist). 

If support for other consoles were to be added, it would likely be consoles that have official support for such features, and even then, I think it would be vastly out of scope of the project.

**Reply by gunnaraas** [reply]
err - I might have misread your question. Were you asking about the ability to play Xbox games from your Switch via Greenlight? 

There's no technical reason it wouldn't work (other than the Switch wifi card being not so good). Judging on past behavior I think it's extremely doubtful that Nintendo would allow something like this to be published on the eShop that allows players to escape their walled garden (though the EU's DMA could force them to allow sideloading regardless). 

There's no reason that you couldn't port this to Switch as homebrew title. The code is all public. I look forward to seeing your pull request!
